src/main/java/org/testng/TestRunner.java

```diff
@@ -178,6 +179,17 @@ public TestRunner(IConfiguration configuration, ISuite suite, XmlTest test,
         suite.getAnnotationFinder(),
         skipFailedInvocationCounts, invokedMethodListeners, classListeners);
   }
+  
+  public TestRunner(IConfiguration configuration, ISuite suite, XmlTest test,
```

Comment:
@krmahadevan, @juherr , why this constructor is deprecated?
as @eyehwan indicates it is used by testng-remote, if there's strong reason for deprecating and removing it, I can add a new adaptor `RemoteTestNG6_13` in testng-remote. otherwise, reverting is a cheap way.

Reply:
@missedone - Nick I am the culprit behind this :( I didnt find any usages of this constructor from within TestNG. So I went ahead and deprecated it in the last release or so. This time I went ahead and cleaned it up. If `testng-remote` is using it, lets restore it. But I would prefer that we add documentation to this constructor stating that it is being used by `testng-remote` and should not be removed. I dont think it makes a lot of sense for us to add a new adaptor, when its easy to restore what was removed. Please let me know if that makes sense.

Reply:
Cool! After back from meeting, everything looks clear already :)

Reply:
@eyehwan , please add javadoc to this constructor as @krmahadevan  suggested. thx

Reply:
Sure, done.

Reply:
Also please add a test that calls this constructor. Removing it should cause a build break.